plugin_settings.py

```diff
@@ -33,15 +39,22 @@ def install():
 
 
 def hook_registry():
-    # On site load, the load function is run for each installed plugin to generate
-    # a list of hooks.
+    """
+    On site load, the load function is run for each installed
+    plugin to generate a list of hooks.
+    """
     return {
-        'publication_fees': {'module': 'plugins.apc.hooks', 'function': 'publication_fees'},
-        'submission_review': {'module': 'plugins.apc.hooks', 'function': 'waiver_info'},
-        'core_article_footer': {'module': 'plugins.apc.hooks', 'function': 'waiver_application'}
+        'publication_fees':
+        {'module': 'plugins.apc.hooks', 'function': 'publication_fees'},
+        'submission_review':
+        {'module': 'plugins.apc.hooks', 'function': 'waiver_info'},
+        'core_article_footer':
+        {'module': 'plugins.apc.hooks', 'function': 'waiver_application'},
```

Comment:
The lack of indentations here feels weird

Reply:
Updated.